File: Programacion/S4/ColaPrueba/src/Clases/Menu.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package Clases;

import java.util.Scanner;

/**
 *
 * @author pqdst
 */
public class Menu {
    
    public static char Opciones()
    {
        System.out.println("Que deseas hacer: ");
        System.out.println("1) Insertar Dato");
        System.out.println("2) Borrar Dato");
        System.out.println("3) Mostrar la Cola");
        System.out.println();
        System.out.println("0) Salir");
        System.out.println();
        System.out.println("Opción : ");
        System.out.println();
        
        Scanner leer = new Scanner (System.in);
        char opc = leer.next().charAt(0);
        return opc;
    }
}

File: Programacion/S4/AppPilaEstaticaC++/Menu.cpp
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */

#include "Pila.h" //porque incluyo a iostream
#include "Menu.h"
//#include <iostream>
//using namespace std;

int Menu::opciones()
{
  int op;
     cout<<"\nTrabajando con una pila : "<<endl;
     cout<<"\n Que deseas hacer : ";
     cout<<"\n 1) Insertar Dato : ";
     cout<<"\n 2) Extraer Dato : ";
     cout<<"\n 3) Salir : ";
     cout<<"\n Opcion : ";
     cin>>op;
  return op;
}


